pkg/apps/application.scss

```diff
@@ -28,6 +28,10 @@
   overflow: hidden;
 }
 
+.ct-missing-metainfo-alert {
+  margin-block-end: 1rem;
```

Comment:
Arg, CSS! :-) What about `<Stack hasGutter>`? But I am far from the authority here.

Reply:
Thanks for the hint! Indeed that's nicer.

I needed to update this again anyway, I found that PackageKit behaved very poorly and blocked the check for a long time. I found out how to do that properly (lightning fast now, and also easier).